docs/advanced-features/using-mdx.md

```diff
@@ -8,6 +8,8 @@ MDX is a superset of markdown that lets you write JSX directly in your markdown
 
 Next.js supports MDX through a number of different means, this page will outline some of the ways you can begin integrating MDX into your Next.js project.
 
+> **Note**: This feature uses extension of Webpack 5. If you use Webpack 4, you can not use @next/mdx.
```

Comment:
I don't think this note is necessary as [webpack 4 has been deprecated in Next.js for a while](https://nextjs.org/docs/upgrading#webpack-4-support-has-been-removed) and if using webpack 4 the older version of this package can be leveraged which should match the Next.js version with webpack 4 still supported.